rust/src/data/ffi.rs

```diff
@@ -139,9 +142,40 @@ pub extern "C" fn opendp_data__slice_as_object(
             .collect::<HashMap<K, V>>();
         Ok(AnyObject::new(map))
     }
+
+    pub fn raw_to_concrete_series(
+        raw: &FfiSlice
+    ) -> Fallible<Series> {
+        let slice = unsafe { slice::from_raw_parts(raw.ptr as *const *const c_void, raw.len) };
+        if slice.len() != 3 {
+            return fallible!(FFI, "Series FfiSlice must have length 3");
+        }
+        // * perma: https://github.com/pola-rs/pyo3-polars/blob/5150d4ca27c287ff4be5cafef243d9a878a8879d/pyo3-polars/src/lib.rs#L117-L126
```

Comment:
Not sure how the linked code relates to this. Add more explanation. If it ever went away, what's the important part?

Reply:
It's no longer relevant!